plugins/woocommerce-admin/client/products/sections/pricing-section.scss

```diff
@@ -4,3 +4,10 @@
 		margin-top: $gap-small;
 	}
 }
+
+.woocommerce-field-width {
+	@media only screen and (max-width: 1032px) {
```

Comment:
We should use the following instead:

```
@include breakpoint( '<960px' ) {
    width: 100%;
}
```

Using the `breakpoint` mixin from https://github.com/woocommerce/woocommerce/blob/112b9ac67a12a86e33d1d9d47027c76f17522083/packages/js/internal-style-build/abstracts/_breakpoints.scss

Since that mixin doesn't support the `1032px` breakpoint, we should check with @jarekmorawski to see what breakpoint we should use (`960px` feels right to me, and is the breakpoint where WP collapses the admin sidebar).